main.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "lib/anagrams"
+
+print "Enter the name of the file you want to check for anagrams? "
+input_text = gets.chomp
```

Comment:
@StefanGajic instead of asking for user input modify the program to get the argument from the CLI, e.g. `ruby main.rb anagrams-wordlist.txt`.

Also please add an option to display only words with anagrams. Currently, the display is too long and the end show something like:

```
puise
tourderie
tourdi
trangr
trangrs
trenness
tui's
volu
volus
vnement
vnements
```

It could be something along the lines of `ruby main.rb anagrams-wordlist.txt anagrams-only`